content/survey_2022.ipynb

```diff
@@ -0,0 +1,4709 @@
+{
```

Comment:
<p>These plots and the description can be very difficult to work through; can it be rewritten more simply? I'm particularly confused by "Among users who mostly use hvPlot, a significant majority of about 92% expressed a preference for hvPlot ", since the plot containing 92% is labeled "Mostly uses HoloViews".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>